PhysicsTools/JetMCAlgos/python/AK4GenJetFlavourInfos_cfi.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+
+ak4GenJetFlavourInfos = cms.EDProducer("JetFlavourClustering",
```

Comment:
it will be easier to maintain if this instead clones from ak4JetFlavourInfos
```python
ak4GenJetFlavourInfos = ak4JetFlavourInfos.clone(jets = "ak4GenJetsNoNu")
```